script.js

```diff
@@ -35,234 +34,235 @@ function insertChar(line, column, char){
     }
 
     document.getElementById('pos'+ line + column).style.backgroundImage = image;
-    marca_posicao[line][column] = char;
+    insert_position[line][column] = char;
```

Comment:
In Portuguese, `marca_posicao` makes sense to the context: it marks the already played positions. However, when translated to the English, the `insert_position` stopped making sense. The name suggests an action (the act of insert something), but this is not what the array is currently doing. The array is not inserting anything, it is just saving the played positions.